docs/adguard-for-android/solving-problems/extending-restricted-settings.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Problems caused by extending restricted settings
+sidebar_position: 18
+---
+
+In an effort to improve user privacy, Google has expanded the restriction settings in Android 15. More settings, such as *App usage access*, now require an additional permission to be enabled. Since the *Firewall* and *Detect Samsung Pay* features require *App usage access* permissions to work properly, you may have experienced problems using them.  To resolve these issues, please follow the steps below:
```

Comment:
we should add links to the respective KB articles (Firewall, Detect Samsung Pay)

Reply:
Also there's an extra space before "To resolve…"

Reply:
There's also some confusion. First, you address "App usage access" as "settings" and later as "permissions"